MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -33,6 +35,14 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		}
 	}
 
+	func display(_ viewModel: FeedErrorViewModel) {
+		guard let message = viewModel.message else {
+			return errorView.hideMessage()
+		}
+
+		errorView.show(message: message)
```

Comment:
Suggestion: 

An `if/else` statement is usually easier to read/follow in such cases:

"If there's a message to show, show it. Else, hide the existing one."

```suggestion
		if let message = viewModel.message {
			errorView.show(message: message)
		} else {
			errorView.hideMessage()}
		}
```

Instead of "guard against not having a message. Else, hide the existing one. Show the message if it passed the guard statement."

Reply:
Ok, I personally try to avoid `if ... else` to avoid extra brackets and make code more straight like in railway oriented programming. I could have used `if` but it will enforce me to get rid of conditional unwrapping like this:

```swift
if viewModel.message == nil {
    return errorView.hideMessage()
}

errorView.show(message: viewModel.message!) // Bang here doesn't look good
```

Reply:
I don't mind if/else if it makes it easier to understand the code. 

If you don't like it, what about inverting it:

```swift
if let message = viewModel.message {
	return errorView.show(message: message)
}

errorView.hideMessage()
```